commands/logout.go

```diff
@@ -105,16 +111,18 @@ func logout(args logoutArguments) error {
 	authHeader := "giantswarm " + args.token
 	logoutResponse, apiResponse, err := apiClient.UserLogout(authHeader, requestIDHeader, logoutActivityName, cmdLine)
 	if err != nil {
-		return fmt.Errorf("Error in API request to logout: %s", err.Error())
+		// special treatment for HTTP 401 (unauthorized) error,
+		// in which case no JSON body is returned.
+		if apiResponse.Response.StatusCode == http.StatusUnauthorized {
+			return microerror.Mask(notAuthorizedError)
+		}
+
+		// other cases
+		return microerror.Mask(errors.New("Error in API request to logout: " + err.Error()))
```

Comment:
According to our guidelines you should create an error variable and mask it. Also we start error messages with lower letter.